yellowbrick/base.py

```diff
@@ -197,18 +197,18 @@ def finalize(self, **kwargs):
 
         Notes
         -----
-        The user calls poof and poof calls finalize. Developers should
+        The user calls show and show calls finalize. Developers should
         implement visualizer-specific finalization methods like setting titles
         or axes labels, etc.
         """
         return self.ax
 
-    def poof(self, outpath=None, clear_figure=False, **kwargs):
+    def show(self, outpath=None, clear_figure=False, **kwargs):
```

Comment:
Very optional: In the interest of backwards compatibility, suggest to retain a `poof()` method for a very limited time that either is an alias for `show()` with deprecation warnings or throws a sensible error saying to use the `show()` method.